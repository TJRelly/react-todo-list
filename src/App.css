* {
    margin: 0;
    padding: 0;
    box-sizing: border-box;
}

.App {
    padding: 3rem 2rem;
    text-align: center;
    background-color: #61dafb;
    height: 100svh;
    display: flex;
    justify-content: center;
}

@media (prefers-reduced-motion: no-preference) {
    .App-link {
        color: #61dafb;
    }
}



