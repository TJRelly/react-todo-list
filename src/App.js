import './App.css';
import TodoList from './components/TodoList';

function App() {
  return (
    <div className="App">
      <TodoList title='My Todos'/>
    </div>
  );
}

export default App;
