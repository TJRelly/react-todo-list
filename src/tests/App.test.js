import { render } from "@testing-library/react";
import App from "../App";

test("renders learn react link", () => {
    render(<App />);
});

test("matches snapshot", function() {
    const {asFragment} = render(<App />);
    expect(asFragment()).toMatchSnapshot();
  });
  
