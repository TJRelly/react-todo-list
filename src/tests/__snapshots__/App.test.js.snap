// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div
      class="TodoList d-flex"
    >
      <form
        class="Todo-form"
      >
        <div
          class="mb-3"
        >
          <label
            class="form-label"
            for="text"
          >
            What do you need todo?
          </label>
          <input
            class="form-control"
            id="text"
            name="text"
            placeholder="Enter a new todo"
            type="string"
            value=""
          />
        </div>
        <button
          class="btn btn-primary"
          type="submit"
        >
          Add
        </button>
      </form>
      <div
        class="TodoList-list"
      >
        <h1>
          My Todos
        </h1>
        <ul />
      </div>
    </div>
  </div>
</DocumentFragment>
`;
